src/systems/ec/implementations/react/packages/language-list/LanguageListItem.jsx

```diff
@@ -15,19 +15,26 @@ const LanguageListItem = ({ label, href, isActive }) => (
         size: 'xs',
       },
     }}
+    hrefLang={lang}
```

Comment:
`hrefLang` and `lang` attributes should be conditional, only appear when the `lang` parameter is a non-empty string

and if `isActive === false`? this is an open question. Is there any added value to adding these attributes if the link points to the current language?

Reply:
I think it doesn't hurt to have it for active item too.
It is less usefull than for other items, but information is still relevant.